crates/sound/src/audio_backends.rs

```diff
@@ -14,7 +14,12 @@ use crate::PCMParams;
 use crate::{Error, Result};
 
 pub trait AudioBackend {
+    fn get_stream_state(&self, _stream_id: u32) -> Result<u32> {
+        Ok(0)
+    }
```

Comment:
Does it make sense to provide a default stream state ? every audio backend should be implementing it no?

Reply:
Also shouldn't the stream state be some enum instead of a random u32?

Reply:
this stream state is different from the audio backend stream state.

Reply:
I think what @bilelmoussaoui means is to removed the generic definition since some backend may rely on it and is wrong.  So that line should only contain the signature.